### PR TITLE
Fix #339: Add ChangeLog placeholder to fix symlink

### LINKED ISSUE #339: There is no changelog any more
docs/changelog.txt is a symbolic link to ../Changelog
Changelog does not exist

### PR DESCRIPTION
**Pull request by houglum**
Running `python setup.py build_sphinx` will result in ChangeLog
being overwritten with actual changes.

### CODE REVIEW

**Comment by berkerpeksag** [comment]
LGTM, thanks!

**Comment by houglum** [comment]
@rbtcollins - any verification needed besides ensuring ChangeLog is overwritten after build?

**Comment by rbtcollins** [comment]
I'm not happy with the manual work that this will reintroduce around version management; like I said in #385 - Requirements:

semver compatible versions for all git revs
clean changelog like currently produced and integrated into docs
clean support for conditional requirements (like we're using today)

tl;dr definitely not going to use manual versioning, life is too short. If you reset this back to the first commit only, which was fine, we can merge it.

**Comment by houglum** [comment]
Yep, I didn't intend to include the second change at the time I issued the pull request.  We added that later after finding that some installation methods for gsutil (downloading via tarball, rather than installing with pip) would not include all necessary dependencies.  Rather than include even more dependencies in the tarball, I just hard-coded the version in my fork we're using.
